app/controllers/StageController.java

```diff
@@ -66,14 +62,12 @@ public StageController(StageRepository stageRepository, RaceRepository raceRepos
         });
     }
 
+    /*
     @BodyParser.Of(BodyParser.Json.class)
     public CompletableFuture<Result> addStage() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)